packages/fast-components-react-msft/src/progress/README.md

```diff
@@ -5,4 +5,7 @@ The *progress* component is an [div](https://developer.mozilla.org/en-US/docs/We
 The *progress* component should be used for an element that displays the progress status for a task that takes a long time or consists of several steps. The *progress* component indicates that the user's request has been received and the application is making progress toward completing the requested action. If the `value` of the *progress* component can't be determined then the indeterminate visuals will be shown until the `value` is determined.
 
 ## Style guidance
-The determinate visuals should indicate to the user how much is remaining of a given task. The indeterminate should be a repeating animation to indicate to the user that their request is being processed and the application is responsive.
\ No newline at end of file
+The determinate visuals should indicate to the user how much is remaining of a given task. The indeterminate should be a repeating animation to indicate to the user that their request is being processed and the application is responsive.

```

Comment:
Maybe "visual display" instead of "visuals" in the first sentence would be clearer?
And if you change that, please add "visual display" to "The indeterminate should be a repeating...", like "The indeterminate visual display should be a repeating...".